src/api/app/views/webui/staging/workflows/new.html.haml

```diff
@@ -8,8 +8,8 @@
       Adding the staging capability to this project will enable you to group
       requests into staging projects and being able to see how they build together.
     %p
-      Managers groups can only be created by an admin. Please contact an
-      = Configuration[:admin_email] == 'unconfigured@openbuildservice.org' ? 'admin' : mail_to(Configuration[:admin_email], 'admin')
+      Managers groups can only be created by an admin. Please create a request via
+      = contact_link
```

Comment:
Before, when `admin_email` had the default value `'unconfigured@openbuildservice.org'` the text admin was rendered, without a link.

After the changes proposed, if  and `contact_url` is not defined, and `admin_email` has the default `'unconfigured@openbuildservice.org'` value, a link to that email is created.

This happens now in the view of creating a project. I think we should fix that creating project case too.